config/02-ce-collector-auth.conf

```diff
@@ -0,0 +1,26 @@
+###############################################################################
+#
+# HTCondor-CE Central Collector authorization configuration
+#
+# EDITS TO THIS FILE ARE PRESERVED UPON UPDATE
+#
+# Modify this file to ban or authorize specific HTCondor-CEs from advertising
+# to this central collector.
+#
+###############################################################################
+
+# To BAN a host, add a line of the form:
+#
+#   DENY_ADVERTISE_SCHEDD = $(DENY_ADVERTISE_SCHEDD), ce.example.com@users.htcondor.org
+#
+# You can have multiple comma-separated hosts in the DENY listing.
+
+# To ADD a custom host, add a line of the form:
+#
+#   COLLECTOR.ALLOW_ADVERTISE_SCHEDD = $(COLLECTOR.ALLOW_ADVERTISE_SCHEDD), ce.example.com@users.htcondor.org/ce.example.com
```

Comment:
Why COLLECTOR.ALLOW_ADVERTISE_SCHEDD instead of ALLOW_ADVERTISE_SCHEDD? Conversely, above, why DENY_ADVERTISE_SCHEDD instead of COLLECTOR.DENY_ADVERTISE_SCHEDD?

Reply:
@bbockelm originally wrote this config but my interpretation is that we only want to allow schedds to advertise to the collector but when we want a host banned, it should be banned completely.